packages/overlay/src/integrations/sentry/components/explore/traces/TraceDetails/components/TraceDetailHeader.tsx

```diff
@@ -9,7 +10,10 @@ export default function TraceDetailHeader({ trace }: TraceDetailHeaderProps) {
   return (
     <div className="border-b-primary-700 bg-primary-950 flex items-center gap-x-2 border-b px-6 py-4">
       <TraceIcon trace={trace} />
-      <h1 className="max-w-full flex-1 truncate text-2xl">{trace.rootTransactionName}</h1>
+      <h1 className="flex w-full flex-1 items-center truncate text-2xl">
+        Trace:&nbsp;&nbsp;
```

Comment:
Do we really need the two `&nbsp;` spaces here? 😅

Reply:
Yeah just to give some space between tag and "Trace:"